src/services/HotelService.java

```diff
@@ -0,0 +1,18 @@
+package services;
+
+import model.hotel.Hotel;
+import model.room.Room;
+
+public class HotelService {
+    private static HotelService instance = new HotelService();
+
+    private HotelService() {}
+
+    public static HotelService getInstance() {
+        return instance;
+    }
+
+    public void addRoom(Room room, Hotel hotel){
```

Comment:
[Fix] cum ar putea un hotel sa adauge mai multe camere? odata ce e creata cladirea, cum isi mai anexeaza alte camere? daca vrei sa zici ca le face disponibile pentru a fi rezervate, eu cred ca aceste camera trebuie sa existe deja in hotel, dar sa aiba un boolean available si aici ai putea avea makeAvailable in loc de addRoom